pinkman-rx3/build.gradle

```diff
@@ -0,0 +1,51 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+
+apply from: "../gradle/version.gradle"
+apply from: '../gradle/publish.gradle'
+
+android {
+    compileSdkVersion build_versions.compile_sdk
+
+    defaultConfig {
+        minSdkVersion build_versions.min_sdk
+        targetSdkVersion build_versions.target_sdk
+
+        versionCode getVersionCodeFromProperties()
+        versionName getVersionNameFromProperties()
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        consumerProguardFiles "consumer-rules.pro"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
```

Comment:
The same as for the coroutines module

Reply:
Minifying is disabled as you can see. But proguard rules are sometimes necessary. For example, it can be useful when some dependency doesn't have its own a rules file.

Reply:
`proguardFiles` will not be included to `aar` to it is useless, isn't it?